cfy_manager/components/usage_collector/scripts/script_utils.py

```diff
@@ -78,8 +66,58 @@ def collect_metadata(data):
     }
 
 
-def send_data(data, url):
-    # for some reason. multi hierarchy dict doesn't pass well to the end point
+def send_data(data, url, interval_type):
+    with get_storage_manager_instance() as sm:
+        usage_collector_info = (sm.list(models.UsageCollector))[0]
+        if interval_type == HOURS_INTERVAL:
+            usage_collector_info.hourly_timestamp = int(time.time())
+        else:
+            usage_collector_info.daily_timestamp = int(time.time())
+        sm.update(usage_collector_info)
     logger.info('The sent data: {0}'.format(data))
     data = {'data': json.dumps(data)}
     post(url, data=data)
+
+
+def try_usage_collector_lock(lock_number):
+    table_name = models.UsageCollector.__tablename__
+    with _get_flask_app().app_context():
+        return storage_utils.try_acquire_lock_on_table(lock_number, table_name)
+
+
+def unlock_usage_collector(lock_number):
+    logger.info('Unlocking usage_collector table')
```

Comment:
low-level stuff like that probably should be `logger.debug`

i'm just worried we'll have useless logs full of "locking.. unlocking.. locking.." etc